aiida_nanotech_empa/parsers/cubegen_pymol_parser.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""AiiDA-Gaussian output parser"""
+from __future__ import absolute_import
+
+import os
+import tempfile
+
+from aiida.orm import FolderData
+
+import aiida_nanotech_empa.utils.pymol_render as pr
+
+from aiida_gaussian.parsers import CubegenBaseParser
+
+
+class CubegenPymolParser(CubegenBaseParser):
+    """Cubegen parser based on pymol"""
+    def _parse_folders(self, retrieved_folder_paths, parser_params):
+
+        if 'isovalues' in parser_params:
+            isovalues = parser_params['isovalues']
+        else:
+            isovalues = [0.01]
+
+        image_folder = tempfile.TemporaryDirectory()
+
+        for folder_path in retrieved_folder_paths:
+            for filename in os.listdir(folder_path):
+                filepath = os.path.join(folder_path, filename)
+
+                if filepath.endswith(".cube"):
+                    for iv in isovalues:
+                        pr.make_pymol_png(filepath,
+                                          isov=iv,
+                                          colors=[(0.9, 0.15, 0.15),
+                                                  (0.0, 0.35, 0.9)],
+                                          output_folder=image_folder.name)
+
+        image_folder_node = FolderData(tree=image_folder.name)
+
+        self.out("cube_image_folder", image_folder_node)
```

Comment:
Can you guarantee that the folder remains after you restart the computer?

Reply:
Yes, I just tested this. The initial temporary folder where I render the images is copied into an aiida node. the temporary directory is deleted after the parser finishes.

Reply:
Sorry, I just don't know how exactly this work: so you create a temporary folder that is populated with images. Further, this folder is used to create the `FolderData` object. So the object which remains is the `FolderData` one, right? The temporary folder will be removed, correct?

Reply:
Yes, exactly right

Reply:
ok, got it, thanks!